src/sbt-test/sbt-1.5/testkit/build.sbt

```diff
@@ -1,22 +1,43 @@
 import _root_.scalafix.sbt.{BuildInfo => Versions}
 
+lazy val scala3InputOutputVersion = "3.0.0"
+
 inThisBuild(
   List(
     scalaVersion := Versions.scala212,
+    crossScalaVersions := List(
+      Versions.scala213,
+      Versions.scala212,
+      Versions.scala211
+    ),
     semanticdbEnabled := true,
     semanticdbVersion := scalafixSemanticdb.revision
   )
 )
 
+crossScalaVersions := Nil
+
 val rules = project
   .disablePlugins(ScalafixPlugin)
   .settings(
     libraryDependencies += "ch.epfl.scala" %% "scalafix-core" % Versions.scalafixVersion
   )
 
 lazy val input = project
+  .settings(
+    crossScalaVersions := List(scala3InputOutputVersion)
+  )
 
 lazy val output = project
+  .settings(
+    crossScalaVersions := List(scala3InputOutputVersion)
+  )
+
+lazy val testsScala3 = project
+  .aggregate(tests)
+  .settings(
+    crossScalaVersions := List(scala3InputOutputVersion)
+  )
```

Comment:
this is very confusing - I am working on a `sbt-projectmatrix` branch variant locally, will push that during the day